src/main/java/DukeCommands.java

```diff
@@ -0,0 +1,54 @@
+public class DukeCommands {
+    public static String spacer = "__________________________________________\n";
+    public static DukeTasks[] list = new DukeTasks[100];
+    static Integer taskLength = 0;
+
+    // Still requires addition of speach lines
+
+    /**
+     * Adds a task with the specified name to a list, and prints a response to the user. 
+     * 
+     * @param taskName Name of task added
+     */
+    public static void addToList(String taskName) {
+        DukeSortInputTask.dukeSortInputTask(list, taskLength, taskName);
+        taskLength += 1;
+        System.out.println("added: " + taskName);
+        System.out.println(spacer);
+    }
+
+    /** Prints a list of all tasks and its marked status to user. */
+    public static void listTasks() {
+        Integer taskIndex = 1;
+        for (DukeTasks task : list) {
+            if (taskIndex > taskLength) {
+                break;
+            }
+            System.out.println(taskIndex + ".[" + task.getTaskType() + "] " + "[" + task.getStatusIcon() + "] " + task.getDescription());
+            taskIndex += 1;
+        }
+        System.out.println(spacer);
+    }
+    
+    /**
+     * Marks the specified task by index number, then prints a response to user. 
+     * 
+     * @param taskIndex Index number of task from list printed by Duke, to be marked
```

Comment:
Good usage of comments between each command's execution which improves readability.